app/models/visit.rb

```diff
@@ -27,7 +27,7 @@ def validate_amount_of_adults
   end
 
   def adult_age
-    AgeValidator.new(Rails.configuration.prison_data[prisoner.prison_name]).adult_age
+    AgeValidator.new(prisoner.prison_name).adult_age
```

Comment:
I feel like this should probably take the prison as a parameter, not do its own lookup.